Source/ui_macosx/ApplicationDelegate.mm

```diff
@@ -100,6 +103,28 @@ -(IBAction)showPreferences: (id)sender
 	[self setupSoundHandler];
 }
 
+-(IBAction)screenCaptureMenuSelected: (id)sender
+{
+	static_cast<CGSH_OpenGL*>(g_virtualMachine->m_ee->m_gs)->OnFlipImplComplete.connect_extended(
```

Comment:
Yeah, we could have a virtual function returning a `Framework::CBitmap` that's overridden in every specific GS handler, with the OpenGL implementation calling `glGetPixels`, etc.